homework/8/api/messages/messages.validations.js

```diff
@@ -0,0 +1,15 @@
+const joi = require('@hapi/joi');
+const { Segments } = require('celebrate');
+
+exports.messagePostValidation = {
+    [Segments.BODY]: {
+        text: joi.string().required()
```

Comment:
```suggestion
        text: joi.string().trim().required()
```
Чтобы не пропустить только пробелы или табуляцию.